src/ui/public/courier/fetch/strategy/__tests__/search.js

```diff
@@ -1,118 +0,0 @@
-import _ from 'lodash';
-import expect from 'expect.js';
-import ngMock from 'ng_mock';
-
-import { SearchStrategyProvider } from '../search';
-import StubIndexPatternProvider from 'test_utils/stub_index_pattern';
-
-describe('SearchStrategyProvider', () => {
```

Comment:
We're losing a lot of tests when just deleting this, one of which would have caught a potential regression.

Reply:
Good call, I'll work on adding these back in.